sdk/src/Core/Amazon.Runtime/Pipeline/Handlers/BaseAuthResolverHandler.cs

```diff
@@ -79,7 +67,7 @@ protected void PreInvoke(IExecutionContext executionContext)
                 if (scheme is AwsV4aAuthScheme || scheme is AwsV4AuthScheme)
```

Comment:
Since we still have SimpleDB using query string signer do we need to update this condition so we use the credentials from the config object for SimpleDB?

Reply:
Nope, SimpleDB does use `AwsV4AuthScheme` and this condition will evaluate to true, however when it reaches the `GetSinger` call, it wont call this method https://github.com/aws/aws-sdk-net/blob/46cbc7042a7b3e78a3aaa6d339176d143539cfb9/sdk/src/Core/Amazon.Runtime/Pipeline/Handlers/BaseAuthResolverHandler.cs#L85 
It will call this overload instead
![image](https://github.com/user-attachments/assets/17b70ab4-7075-45a4-86cf-1518efbbbb87)

Similar to how S3 calls this overload to get the `S3Signer`
https://github.com/aws/aws-sdk-net/blob/46cbc7042a7b3e78a3aaa6d339176d143539cfb9/sdk/src/Services/S3/Generated/Internal/AmazonS3AuthResolver.cs#L81

This method is added if `DetermineSignerOverride` doesn't return null for the `SignatureVersion`
https://github.com/aws/aws-sdk-net/blob/46cbc7042a7b3e78a3aaa6d339176d143539cfb9/generator/ServiceClientGeneratorLib/Generators/AuthResolvers/ModeledResolver.tt#L76